src/js/tracker.js

```diff
@@ -348,7 +334,14 @@
 			preservePageViewId = false,
 
 			// Whether first trackPageView was fired and pageViewId should not be changed anymore until reload
-			pageViewSent = false;
+			pageViewSent = false,
+
+			// Activity tracking config for callback and pacge ping variants
+			activityTrackingConfig = {
+				enabled: false,
+				installed: false, // Guard against installing the activity tracker more than once per Tracker instance
+				configurations: {}
```

Comment:
I like this 👍

But.... I think you never use the object key for anything.  Therefore, why not store it as a plain array instead of a key-val dictionary?

This has a nice side effect that the client can call `enableActivityTrackingCallback` as many times as they want with different callback functions.

You might prefer to drop `enabled` and instead do a quick test for if the array is empty.

Reply:
I did this first (using an array) but enableActivityCallback has always worked in such a way that you can call it multiple times and it adjusts the previous version. I feared adding multiple versions would either lead to bugs, cause some sites to send duplicated page pings or make the code more complicated by only allowing 1 page ping version but multiple callback versions. 
My thinking ultimately ended up as allowing a single version of each was safer and more consistent.

Reply:
Oh, forgot to talk about .configurations. 
I made this seperate so when I iterate over the keys, I don't have to check if I'm looking at a callback configuration or one of the activityTrackingConfig booleans. Maybe there is another way of tidying this up but without multiple objects I'm not sure what would be neater.